fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -38,7 +40,7 @@
     private Map<Column, Set<Interval>> workingIntervals;
     private final Object mutex = new Object();
     private final Map<String, String> logicalToPhysicalColumnNames;
-    private final Map<String, String> physicalToLogicalColumnNames;
+    private final Map<String, List<String>> physicalToLogicalColumnNames;
```

Comment:
I don't think we want to have any headaches around nested types.  Let's just use set for now.